central/scannerdefinitions/handler/handler.go

```diff
@@ -97,7 +90,11 @@ var (
 		"name2repos": "repomapping/container-name-repos-map.json",
 		"repo2cpe":   "repomapping/repository-to-cpe.json",
 	}
-	minorVersionPattern = regexp.MustCompile(`^\d+\.\d+`)
+
+	mainVersionPattern = regexp.MustCompile(`^\d+\.\d+\.\d+`)
+
+	// mainVersionNum is the version number of central's version tag.
+	mainVersionNum string
```

Comment:
Kinda, if you look at https://semver.org/ the suffixes are part of the semantic versioning. What we are parsing is strictly referred to as "core version", but I thought that "version numbers" were more generic and less confusing. Let me know if you have any other suggestion, otherwise I leave it.